index.js

```diff
@@ -1,99 +1,89 @@
-/* eslint-disable no-unused-vars */
-let books = [];
-
-const generateId = () => {
-  let id = Math.ceil(Math.random() * 100000000000);
-  while (books.indexOf(id) !== -1) {
-    id = Math.ceil(Math.random() * 100000000000);
+class Methods {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
   }
-  return id;
-};
 
-const addBook = (title, author) => {
-  const awesomeBook = {
-    title,
-    author,
-    id: generateId(),
+  addBook = (bookTitle, bookAuthor, bookId) => {
+    const book = {
+      title: bookTitle,
+      author: bookAuthor,
+      id: bookId,
+    };
+    this.books.push(book);
   };
-  books.push(awesomeBook);
-};
 
-const removeBook = (id) => {
-  books = books.filter((book) => book.id !== id);
-};
+  removeBook = (id) => {
+    this.books.splice(this.books[id - 1], 1);
+  };
+}
 
-const showBooks = () => {
-  books.forEach((book) => {
-    console.log(`${book.title} by ${book.author}`);
-  });
-};
+const methods = new Methods();
+methods.books = [];
 
```

Comment:
You did a great job implementing your classes and using your array of objects from within the class 🔥